js/functions.js

```diff
@@ -568,28 +610,23 @@ function myDebounce(cb, d) {
     }, d);
   };
 }
-const dynamicStyleDrawerWithDebounce = myDebounce((drawerContent, rect) => {
-  dynamicStyleDrawer(drawerContent, rect);
-}, 1000);
+export const dynamicStyleDrawerWithDebounce = myDebounce(
+  (drawerContent, rect) => {
+    dynamicStyleDrawer(drawerContent, rect);
+  },
+  1000
+);
 
-// let timer;
-// function dynamicStyleDrawerWithDebounce(fn, d, ...args) {
-//   clearTimeout(timer);
-//   timer = setTimeout(() => {
-//     fn(...args);
-//   }, d);
-// }
-
-function drawerShow(drawerContent) {
+export function drawerShow(drawerContent) {
   drawerContent.style.display = "inline";
 }
 
-function drawerClose(drawerContent) {
+export function drawerClose(drawerContent) {
   drawerContent.style.display = "none";
 }
 
 //show the stored nums with child append
-function showStoredNumbers() {
+export function showStoredNumbers() {
   const arrayOfNumbers = getValueFromLocal("storedNums")?.split(",");
   if (arrayOfNumbers === undefined) {
     document.getElementById("empty-msg").innerText =
```

Comment:
Please store this node in a constant instead of querying it more than once.

Reply:
stored in the div and then using it .